src/sentry/feedback/endpoints/feedback_ingest.py

```diff
@@ -0,0 +1,154 @@
+            "sdk": data["sdk"],
+            "user": data["user"],
+            "request": data["request"],
+        }
+        ret["date_added"] = datetime.datetime.fromtimestamp(data["timestamp"])
+        ret["feedback_id"] = data["event_id"]
+        ret["url"] = ""
+        ret["project_id"] = self.context["project"].id
+        ret["replay_id"] = ""
+        ret["message"] = ""
+
+        return ret
+
+
+class ProjectMonitorPermission(ProjectPermission):
+    scope_map = {
+        "GET": ["project:read", "project:write", "project:admin"],
+        "POST": ["project:read", "project:write", "project:admin"],
+        "PUT": ["project:read", "project:write", "project:admin"],
+        "DELETE": ["project:read", "project:write", "project:admin"],
+    }
+
+
+@region_silo_endpoint
+class FeedbackIngestEndpoint(Endpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.FEEDBACK
+
+    authentication_classes = (
+        DSNAuthentication,
+        TokenAuthentication,
+        OrgAuthTokenAuthentication,
+        ApiKeyAuthentication,
+    )
+
+    permission_classes = (ProjectMonitorPermission,)
+
+    def convert_args(
+        self,
+        request: Request,
+        organization_slug: str | None = None,
+        *args,
+        **kwargs,
+    ):
+        using_dsn_auth = isinstance(request.auth, ProjectKey)
+
+        # When using DSN auth we're able to infer the organization slug
+        if not organization_slug and using_dsn_auth:
+            organization_slug = request.auth.project.organization.slug  # type: ignore
+
+        if organization_slug:
+            try:
+                organization = Organization.objects.get_from_cache(slug=organization_slug)
+                # Try lookup by slug first. This requires organization context since
+                # slugs are unique only to the organization
+            except (Organization.DoesNotExist):
+                pass
+
+        project = request.auth.project  # type: ignore
+
+        if project.status != ObjectStatus.ACTIVE:
+            raise ResourceDoesNotExist
+
+        if using_dsn_auth and project.id != request.auth.project_id:  # type: ignore
+            raise ResourceDoesNotExist
+
+        if organization_slug and project.organization.slug != organization_slug:
+            raise ResourceDoesNotExist
+
+        # Check project permission. Required for Token style authentication
+        self.check_object_permissions(request, project)
+
+        with configure_scope() as scope:
+            scope.set_tag("project", project.id)
+
+        bind_organization_context(project.organization)
+
+        request._request.organization = project.organization  # type: ignore
+
+        kwargs["organization"] = organization
+        kwargs["project"] = project
+        return args, kwargs
+
+    def get(self, request: Request, organization: Organization, project: Project) -> Response:
```

Comment:
we can remove the `get` method here since we're only gonna have this endpoint do ingestion.